app/views/account.html

```diff
@@ -1,18 +1,23 @@
-
 <div ng-attr-id="{{data.id_str}}" class="home-user-info">
     <div class="blue-background-placeholder">
         <a ng-href="{{'./search?q=from:' + data.screen_name}}" ><img ng-src="{{data.profile_banner_url }}" alt="user-banner-photo" ng-show="data.profile_banner_url" fallback-src="/images/anon_400x400.png"></a>
     </div>
     <div class="home-user-info-content">
         <div class="top-content">
-            <a ng-href="{{'./search?q=from:' + data.screen_name}}"><img fallback-src="/images/anon_200x200.png" ng-src="{{data.profile_image_url_https | homePageImage }}"></a>
+            <div class="account-image-and-following-text">
```

Comment:
Just added to combine things in a div. After "following" button is introduced. @sudheesh001 
No CSS as of yet. Can remove class.